Formula/ddcctl.rb

```diff
@@ -0,0 +1,16 @@
+class Ddcctl < Formula
+  desc "DDC monitor controls (brightness) for Mac OSX command-line"
+  homepage "https://github.com/kfix/ddcctl"
+  url "https://github.com/kfix/ddcctl/archive/refs/tags/v0.tar.gz"
+  sha256 "8440f494b3c354d356213698dd113003245acdf667ed3902b0d173070a1a9d1f"
+  license :public_domain
+
+  def install
+    system "make"
+    bin.install "#{buildpath}/bin/release/ddcctl"
+  end
+
+  test do
+    assert_match "found 0 external display", shell_output("#{bin}/ddcctl -d 1 -b 100 2>/dev/null | cat")
```

Comment:
```suggestion
    assert_match "found 0 external display", shell_output("#{bin}/ddcctl -d 1 -b 100")
```
Why do you need ` 2>/dev/null | cat`? You most likely can do without it, possibly by using `pipe_output` instead of `shell_output`.